challenge-14/submissions/atplay90/solution-template.go

```
@@ -0,0 +1,522 @@
+		user, err := userServer.GetUser(r.Context(), userID)
+		if err != nil {
+			if status.Code(err) == codes.NotFound {
+				http.Error(w, err.Error(), http.StatusNotFound)
+			} else {
+				http.Error(w, err.Error(), http.StatusInternalServerError)
+			}
+			return
+		}
+
+		w.Header().Set("Content-Type", "application/json")
+		json.NewEncoder(w).Encode(user)
```

golangci-lint (2.12.2)

[error] Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)